Talk won't compile with latest master

I can successfully run spreed master on server master locally, but it won't work with this images. Any idea?

Command run:
```
sudo docker run -it \
-e SERVER_BRANCH=master \
-e TALK_BRANCH=master \
--name nextcloud-easy-test-master \
-p 127.0.0.1:8443:443 ghcr.io/szaimen/nextcloud-easy-test:latest
```

Result
```
Cloning into 'spreed'...
remote: Enumerating objects: 1176, done.
remote: Counting objects: 100% (1176/1176), done.
remote: Compressing objects: 100% (959/959), done.
remote: Total 1176 (delta 380), reused 421 (delta 174), pack-reused 0
Receiving objects: 100% (1176/1176), 16.73 MiB | 4.78 MiB/s, done.
Resolving deltas: 100% (380/380), done.
+ '[' -f package.json ']'
++ grep '"node":' package.json
++ head -1
+ local 'NODE_LINE=             "node": "^14.0.0",'
+ '[' -n '              "node": "^14.0.0",' ']'
+ echo '                "node": "^14.0.0",'
+ grep -q '>='
+ set +x
Now using node v14.19.3 (npm v8.13.1)
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
Package operations: 1 install, 0 updates, 0 removals
  - Downloading firebase/php-jwt (v6.2.0)
  - Installing firebase/php-jwt (v6.2.0): Extracting archive
Package webmozart/path-util is abandoned, you should avoid using it. Use symfony/filesystem instead.
Generating optimized autoload files
npm notice
npm notice New minor version of npm available! 8.13.1 -> 8.14.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.14.0
npm notice Run npm install -g npm@8.14.0 to update!
npm notice
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @vue/cli-plugin-unit-jest@5.0.8
npm ERR! Found: @vue/vue2-jest@28.0.1
npm ERR! node_modules/@vue/vue2-jest
npm ERR!   dev @vue/vue2-jest@"^28.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional @vue/vue2-jest@"^27.0.0-alpha.3" from @vue/cli-plugin-unit-jest@5.0.8
npm ERR! node_modules/@vue/cli-plugin-unit-jest
npm ERR!   dev @vue/cli-plugin-unit-jest@"^5.0.8" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @vue/vue2-jest@27.0.0
npm ERR! node_modules/@vue/vue2-jest
npm ERR!   peerOptional @vue/vue2-jest@"^27.0.0-alpha.3" from @vue/cli-plugin-unit-jest@5.0.8
npm ERR!   node_modules/@vue/cli-plugin-unit-jest
npm ERR!     dev @vue/cli-plugin-unit-jest@"^5.0.8" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /var/www/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /var/www/.npm/_logs/2022-07-17T20_52_42_225Z-debug-0.log
Could not compile the spreed app.
```

Check for updates image:
```
latest: Pulling from szaimen/nextcloud-easy-test
Digest: sha256:003116cc3740ce684bba31bfd981a4d8594cb63ad377557b221c228a5f187daa
Status: Image is up to date for ghcr.io/szaimen/nextcloud-easy-test:latest
ghcr.io/szaimen/nextcloud-easy-test:latest
```

--- a/remote.sh
+++ b/remote.sh
@@ -54,6 +54,25 @@ handle_node_version() {
     fi
 }
 
+# Handle npm versions
+handle_npm_version() {
+    set -x
+    if [ -f package.json ]; then
+        local NPM_LINE=$(grep '"npm":' package.json | head -1)
+    fi
+    if [ -n "$NPM_LINE" ] && echo "$NPM_LINE" | grep -q '\^'; then
+        local NPM_VERSION="$(echo "$NPM_LINE" | grep -oP '\^[0-9]+' | sed 's|\^||')"
+        if [ -n "$NPM_VERSION" ] && [ "$NPM_VERSION" -eq 7 ]; then
+            set +x
+            npm i -g npm@latest-7
+            return
+        fi
+    fi
+
+    set +x
+    nvm install-latest-npm
+}
+
 # Handle empty server branch variable
 if [ -z "$SERVER_BRANCH" ]; then
     export SERVER_BRANCH="nextcloud:master"
@@ -90,6 +109,10 @@ if ! [ -f /var/www/server-completed ]; then
         if version_greater "$installed_version" "24.0.0.0"; then
             # Handle node version
             handle_node_version
+
+            # Handle npm version
+            handle_npm_version
+
             echo "Compiling server..."
             if ! npm ci || ! npm run build --if-present; then
                 echo "Could not compile server."
@@ -199,6 +222,9 @@ if [ -n "$BRANCH" ] && ! [ -f "/var/www/$APPID-completed" ]; then
     
     # Handle node version
     handle_node_version
+
+    # Handel npm version
+    handle_npm_version
 
     # if [ "$APPID" = mail ]; then
     #     wget https://getcomposer.org/download/1.10.22/composer.phar

